src/Components/Navigation.js

```diff
@@ -0,0 +1,91 @@
+import React from 'react';
+import '../Navigation.css';
+import { Link } from 'react-router-dom';
+
+function Navigation() {
```

Comment:
- Please, consider using arrow funcs according to Javascript ES6 good practices.